examples/notebooks/Getting Started/Tutorial 2 - Compare models.ipynb

```diff
@@ -0,0 +1,147 @@
+    "\n",
+    "Again, the first step is to import the pybamm library into the notebook:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install pybamm -q    # install PyBaMM if it is not installed\n",
+    "import pybamm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We start creating a list of all the models we wish to solve"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "models = [\n",
+    "    pybamm.lithium_ion.SPM(),\n",
+    "    pybamm.lithium_ion.SPMe(),\n",
+    "    pybamm.lithium_ion.DFN(),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "and now we can loop over the list, creating and solving simulations as we go. The solved simulations are stored in the list `sims`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sims = []\n",
+    "for model in models:\n",
+    "    sim = pybamm.Simulation(model)\n",
+    "    sim.solve()\n",
+    "    sims.append(sim)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now pass our list of simulations to the dynamic plot method, which will plot the different outputs in the same figure"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "63e7fa6511714626abc1173c35aad054",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "interactive(children=(FloatSlider(value=0.0, description='t', max=3599.9999999999995, step=35.99999999999999),…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<pybamm.plotting.quick_plot.QuickPlot at 0x7f089bff9518>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pybamm.dynamic_plot(sims, time_unit=\"seconds\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial we have seen how easy it is to run and compare different electrochemical models. In [Tutorial 3](./Tutorial%203%20-%20Basic%20plotting.ipynb) we cover how to simulate and compare different models."
```

Comment:
this link is broken. should capital P is plotting or change to lowercase in the notebook name

Reply:
I have fixed it now, and also fixed the links in Tutorials 6 and 7.